src/net-questdb-client/net-questdb-client.csproj

```diff
@@ -14,12 +14,18 @@
         <RepositoryUrl>https://github.com/questdb/net-questdb-client</RepositoryUrl>
         <PackageTags>QuestDB, ILP</PackageTags>
         <Company>QuestDB Limited</Company>
-        <TargetFramework>netstandard2.1</TargetFramework>
         <PackageVersion>1.1.2</PackageVersion>
+        <TargetFramework>net6.0</TargetFramework>
```

Comment:
Please stick to netstandard2.1. This client does not need many features, all should be available in netstandard. It gives broader compatibility.